packages/client/CHANGELOG.md

```diff
@@ -2,6 +2,16 @@
 
 ## Unreleased
 
+### Fixed
+
+- Properly decode metadata v14 for chains that don't support metadata v15
+
+## 1.5.2 - 2024-10-16
+
+### Fixed
+
+- Missing dependency `@polkadot-api/ink-contracts`
+
```

Comment:
This was already released in `1.5.1`, not sure why this CHANGELOG is re-added.

Reply:
This is obviously a mistake... I don't know how it happened, but let me remove this. Thanks!